<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>800</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>800</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(/home/iot/바탕화면/12/1234.png);
</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>800</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* QFrame의 배경 스타일 */
QFrame {
    background-color: #ffffff; /* 배경 색상 */
    border: 1px solid #cccccc; /* 테두리 스타일 및 색상 */
    border-radius: 4px; /* 테두리 모서리 둥글게 설정 */
    padding: 5px; /* 내부 여백 설정 */
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2013</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2014</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2022</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton7">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>490</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* QPushButton의 스타일 */
QPushButton {
    background-color: #3498db; /* 배경 색상 */
    color: #ffffff; /* 글꼴 색상 */
    border: 1px solid #2980b9; /* 테두리 스타일 및 색상 */
    border-radius: 4px; /* 테두리 모서리 둥글게 설정 */
    padding: 5px 10px; /* 내부 여백 설정 */
    font-size: 14px; /* 글꼴 크기 */
}

/* QPushButton에 마우스를 올렸을 때의 스타일 (마우스 오버 효과) */
QPushButton:hover {
    background-color: #2980b9; /* 마우스 오버 시 배경 색상 변경 */
    border: 1px solid #3498db; /* 마우스 오버 시 테두리 스타일 변경 */
}
</string>
      </property>
      <property name="text">
       <string>광주</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton8">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>490</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* QPushButton의 스타일 */
QPushButton {
    background-color: #3498db; /* 배경 색상 */
    color: #ffffff; /* 글꼴 색상 */
    border: 1px solid #2980b9; /* 테두리 스타일 및 색상 */
    border-radius: 4px; /* 테두리 모서리 둥글게 설정 */
    padding: 5px 10px; /* 내부 여백 설정 */
    font-size: 14px; /* 글꼴 크기 */
}

/* QPushButton에 마우스를 올렸을 때의 스타일 (마우스 오버 효과) */
QPushButton:hover {
    background-color: #2980b9; /* 마우스 오버 시 배경 색상 변경 */
    border: 1px solid #3498db; /* 마우스 오버 시 테두리 스타일 변경 */
}
</string>
      </property>
      <property name="text">
       <string>제주</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton6">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>290</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* QPushButton의 스타일 */
QPushButton {
    background-color: #3498db; /* 배경 색상 */
    color: #ffffff; /* 글꼴 색상 */
    border: 1px solid #2980b9; /* 테두리 스타일 및 색상 */
    border-radius: 4px; /* 테두리 모서리 둥글게 설정 */
    padding: 5px 10px; /* 내부 여백 설정 */
    font-size: 14px; /* 글꼴 크기 */
}

/* QPushButton에 마우스를 올렸을 때의 스타일 (마우스 오버 효과) */
QPushButton:hover {
    background-color: #2980b9; /* 마우스 오버 시 배경 색상 변경 */
    border: 1px solid #3498db; /* 마우스 오버 시 테두리 스타일 변경 */
}
</string>
      </property>
      <property name="text">
       <string>강원</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* QPushButton의 스타일 */
QPushButton {
    background-color: #3498db; /* 배경 색상 */
    color: #ffffff; /* 글꼴 색상 */
    border: 1px solid #2980b9; /* 테두리 스타일 및 색상 */
    border-radius: 4px; /* 테두리 모서리 둥글게 설정 */
    padding: 5px 10px; /* 내부 여백 설정 */
    font-size: 14px; /* 글꼴 크기 */
}

/* QPushButton에 마우스를 올렸을 때의 스타일 (마우스 오버 효과) */
QPushButton:hover {
    background-color: #2980b9; /* 마우스 오버 시 배경 색상 변경 */
    border: 1px solid #3498db; /* 마우스 오버 시 테두리 스타일 변경 */
}
</string>
      </property>
      <property name="text">
       <string>충북</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* QPushButton의 스타일 */
QPushButton {
    background-color: #3498db; /* 배경 색상 */
    color: #ffffff; /* 글꼴 색상 */
    border: 1px solid #2980b9; /* 테두리 스타일 및 색상 */
    border-radius: 4px; /* 테두리 모서리 둥글게 설정 */
    padding: 5px 10px; /* 내부 여백 설정 */
    font-size: 14px; /* 글꼴 크기 */
}

/* QPushButton에 마우스를 올렸을 때의 스타일 (마우스 오버 효과) */
QPushButton:hover {
    background-color: #2980b9; /* 마우스 오버 시 배경 색상 변경 */
    border: 1px solid #3498db; /* 마우스 오버 시 테두리 스타일 변경 */
}
</string>
      </property>
      <property name="text">
       <string>대전</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>110</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* QPushButton의 스타일 */
QPushButton {
    background-color: #3498db; /* 배경 색상 */
    color: #ffffff; /* 글꼴 색상 */
    border: 1px solid #2980b9; /* 테두리 스타일 및 색상 */
    border-radius: 4px; /* 테두리 모서리 둥글게 설정 */
    padding: 5px 10px; /* 내부 여백 설정 */
    font-size: 14px; /* 글꼴 크기 */
}

/* QPushButton에 마우스를 올렸을 때의 스타일 (마우스 오버 효과) */
QPushButton:hover {
    background-color: #2980b9; /* 마우스 오버 시 배경 색상 변경 */
    border: 1px solid #3498db; /* 마우스 오버 시 테두리 스타일 변경 */
}
</string>
      </property>
      <property name="text">
       <string>대구</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* QPushButton의 스타일 */
QPushButton {
    background-color: #3498db; /* 배경 색상 */
    color: #ffffff; /* 글꼴 색상 */
    border: 1px solid #2980b9; /* 테두리 스타일 및 색상 */
    border-radius: 4px; /* 테두리 모서리 둥글게 설정 */
    padding: 5px 10px; /* 내부 여백 설정 */
    font-size: 14px; /* 글꼴 크기 */
}

/* QPushButton에 마우스를 올렸을 때의 스타일 (마우스 오버 효과) */
QPushButton:hover {
    background-color: #2980b9; /* 마우스 오버 시 배경 색상 변경 */
    border: 1px solid #3498db; /* 마우스 오버 시 테두리 스타일 변경 */
}
</string>
      </property>
      <property name="text">
       <string>서울</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* QPushButton의 스타일 */
QPushButton {
    background-color: #3498db; /* 배경 색상 */
    color: #ffffff; /* 글꼴 색상 */
    border: 1px solid #2980b9; /* 테두리 스타일 및 색상 */
    border-radius: 4px; /* 테두리 모서리 둥글게 설정 */
    padding: 5px 10px; /* 내부 여백 설정 */
    font-size: 14px; /* 글꼴 크기 */
}

/* QPushButton에 마우스를 올렸을 때의 스타일 (마우스 오버 효과) */
QPushButton:hover {
    background-color: #2980b9; /* 마우스 오버 시 배경 색상 변경 */
    border: 1px solid #3498db; /* 마우스 오버 시 테두리 스타일 변경 */
}
</string>
      </property>
      <property name="text">
       <string>경기도</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>490</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* QPushButton의 스타일 */
QPushButton {
    background-color: #3498db; /* 배경 색상 */
    color: #ffffff; /* 글꼴 색상 */
    border: 1px solid #2980b9; /* 테두리 스타일 및 색상 */
    border-radius: 4px; /* 테두리 모서리 둥글게 설정 */
    padding: 5px 10px; /* 내부 여백 설정 */
    font-size: 14px; /* 글꼴 크기 */
}

/* QPushButton에 마우스를 올렸을 때의 스타일 (마우스 오버 효과) */
QPushButton:hover {
    background-color: #2980b9; /* 마우스 오버 시 배경 색상 변경 */
    border: 1px solid #3498db; /* 마우스 오버 시 테두리 스타일 변경 */
}
</string>
      </property>
      <property name="text">
       <string>부산</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton9">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>650</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* QPushButton의 스타일 */
QPushButton {
    background-color: #3498db; /* 배경 색상 */
    color: #ffffff; /* 글꼴 색상 */
    border: 1px solid #2980b9; /* 테두리 스타일 및 색상 */
    border-radius: 4px; /* 테두리 모서리 둥글게 설정 */
    padding: 5px 10px; /* 내부 여백 설정 */
    font-size: 14px; /* 글꼴 크기 */
}

/* QPushButton에 마우스를 올렸을 때의 스타일 (마우스 오버 효과) */
QPushButton:hover {
    background-color: #2980b9; /* 마우스 오버 시 배경 색상 변경 */
    border: 1px solid #3498db; /* 마우스 오버 시 테두리 스타일 변경 */
}
</string>
      </property>
      <property name="text">
       <string>전주</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QFrame" name="frame2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>800</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* QFrame의 배경 스타일 */
QFrame {
    background-color: #ffffff; /* 배경 색상 */
    border: 1px solid #cccccc; /* 테두리 스타일 및 색상 */
    border-radius: 4px; /* 테두리 모서리 둥글게 설정 */
    padding: 5px; /* 내부 여백 설정 */
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>569</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* QLabel의 일반 상태 스타일 */
QLabel {
    color: #333333; /* 텍스트 색상 */
    background-color: transparent; /* 배경색상 투명 */
    font-size: 16px; /* 글꼴 크기 */
    font-weight: bold; /* 글꼴 두껍게 설정 */
}

/* QLabel의 마우스 호버 상태 스타일 */
QLabel:hover {
    color: #555555; /* 호버 상태에서의 텍스트 색상 */
}

/* QLabel의 클릭 상태 스타일 */
QLabel:pressed {
    color: #777777; /* 클릭 상태에서의 텍스트 색상 */
}
</string>
      </property>
      <property name="text">
       <string>최소</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>669</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* QLabel의 일반 상태 스타일 */
QLabel {
    color: #333333; /* 텍스트 색상 */
    background-color: transparent; /* 배경색상 투명 */
    font-size: 16px; /* 글꼴 크기 */
    font-weight: bold; /* 글꼴 두껍게 설정 */
}

/* QLabel의 마우스 호버 상태 스타일 */
QLabel:hover {
    color: #555555; /* 호버 상태에서의 텍스트 색상 */
}

/* QLabel의 클릭 상태 스타일 */
QLabel:pressed {
    color: #777777; /* 클릭 상태에서의 텍스트 색상 */
}
</string>
      </property>
      <property name="text">
       <string>평균</string>
      </property>
     </widget>
     <widget class="QLabel" name="max_label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>619</y>
        <width>100</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>550</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* QLabel의 일반 상태 스타일 */
QLabel {
    color: #333333; /* 텍스트 색상 */
    background-color: transparent; /* 배경색상 투명 */
    font-size: 16px; /* 글꼴 크기 */
    font-weight: bold; /* 글꼴 두껍게 설정 */
}

/* QLabel의 마우스 호버 상태 스타일 */
QLabel:hover {
    color: #555555; /* 호버 상태에서의 텍스트 색상 */
}

/* QLabel의 클릭 상태 스타일 */
QLabel:pressed {
    color: #777777; /* 클릭 상태에서의 텍스트 색상 */
}
</string>
      </property>
      <property name="text">
       <string>상품</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>619</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* QLabel의 일반 상태 스타일 */
QLabel {
    color: #333333; /* 텍스트 색상 */
    background-color: transparent; /* 배경색상 투명 */
    font-size: 16px; /* 글꼴 크기 */
    font-weight: bold; /* 글꼴 두껍게 설정 */
}

/* QLabel의 마우스 호버 상태 스타일 */
QLabel:hover {
    color: #555555; /* 호버 상태에서의 텍스트 색상 */
}

/* QLabel의 클릭 상태 스타일 */
QLabel:pressed {
    color: #777777; /* 클릭 상태에서의 텍스트 색상 */
}
</string>
      </property>
      <property name="text">
       <string>최대</string>
      </property>
     </widget>
     <widget class="QLabel" name="min_label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>569</y>
        <width>100</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>491</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="graph_verticalLayout"/>
     </widget>
     <widget class="QLabel" name="average_label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>669</y>
        <width>100</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton13">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
    background-color: #3498db;
    color: #ffffff;
    padding: 10px 20px;
    font-size: 16px;
    border-radius: 5px;</string>
      </property>
      <property name="text">
       <string>뒤로가기</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>580</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>열무</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>얼갈이배추</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>물오징어</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>명태</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>갈치</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>고등어</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>방울토마토</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>감귤</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>단감</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>바나나</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>참다래</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>팽이버섯</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>사과</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>배</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>포도</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>감귤</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>파</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>생강</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>미나리</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>깻잎</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>피망(청)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>느타리버섯</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>호박</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>토마토</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>무</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>당근</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
